src/lib/output/themes/default/DefaultTheme.tsx

```diff
@@ -214,7 +225,7 @@ export class DefaultTheme extends Theme {
             urls.push(new UrlMapping("index.html", project, this.indexTemplate));
         }
 
-        if (getHierarchyRoots(project).length) {
+        if (this.application.options.getValue("includeHierarchySummary") && getHierarchyRoots(project).length) {
```

Comment:
I admit it might be rare, but I've had a small project that had a single class that involved a hierarchy. Having an entire page just for two lines that already appeared on both of the linked pages seemed like bloat to me.